bayesian/utils.py

```diff
@@ -65,20 +66,22 @@ def get_user_email(user_profile):
 
 def create_component_bookkeeping(ecosystem, packages_list, request_args, headers):
     """Run the component analysis for given ecosystem+package+version."""
-    args = {
-        'external_request_id': headers.get('X-Request-Id', None),
-        'data': {
-            'api_name': 'component_analyses_post',
-            'manifest_hash': request_args.get('utm_content', None),
-            'ecosystem': ecosystem,
-            'packages_list': packages_list,
-            'user_id': headers.get('uuid', None),
-            'user_agent': headers.get('User-Agent', None),
-            'source': request_args.get('utm_source', None),
-            'telemetry_id': headers.get('X-Telemetry-Id', None)
+    payload = {
+        "external_request_id": headers.get('X-Request-Id', None),
+        "flowname": "componentApiFlow",
+        "data": {
+            "api_name": "component_analyses_post",
+            "manifest_hash": request_args.get('utm_content', None),
+            "ecosystem": ecosystem,
+            "packages_list": packages_list,
+            "user_id": headers.get('uuid', None),
+            "user_agent": headers.get('User-Agent', None),
+            "source": request_args.get('utm_source', None),
+            "telemetry_id": headers.get('X-Telemetry-Id', None)
         }
     }
-    return server_run_flow('componentApiFlow', args)
+
+    trigger_workerflow(payload)
```

Comment:
This can throw exception, so better to use try except to avoid any break in flow.

Reply:
If any exception is thrown wont it be inside jobs api or utils.where this has been handeled with try except.